velox/docs/functions/presto/datetime.rst

```diff
@@ -24,6 +24,10 @@ Date and Time Functions
 
     Returns ``timestamp`` as a UNIX timestamp.
 
+.. function:: current_time() -> string
```

Comment:
The functions are added in the documentation in alphabetical order. Please add this function in the correct order in this file.

Reply:
@aditi-pandit Moved it to where it should be in alphabetical order